log/log.go

```diff
@@ -39,10 +41,63 @@ func (f levelFlag) Set(level string) error {
 	return nil
 }
 
+func setFormatterSylog(appname, local string) error {
```

Comment:
"Sylog" -> "Syslog"

This function name (`setFormatterSyslog`) is inconsistent with the other one (`setJsonFormatter`). Maybe rename it to `setSyslogFormatter`?